npm/ls-lint/postinstall.js

```diff
@@ -0,0 +1,24 @@
+const { spawn } = require("child_process");
+const { join } = require("path");
+
+const isCI = process.env.CI;
+
+if (!isCI) {
+  if (process.platform === 'darwin') {
+    binary = 'ls-lint-darwin';
+  } else if (process.platform === 'linux') {
+    binary = 'ls-lint-linux';
+  } else if (process.platform === 'win32') {
+    binary = 'ls-lint-windows.exe';
+  } else {
+    console.log("Unsupported OS");
```

Comment:
we should use a error message like: 

console.log('ls-lint builds are not available on platform: ' + process.platform)